packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4487,9 +4461,59 @@ export default class Meeting extends StatelessWebexPlugin {
         );
 
         joinFailed(error);
+
         this.deferJoin = undefined;
 
         return Promise.reject(error);
+      })
+      .then((join) => {
+        // @ts-ignore - config coming from registerPlugin
+        if (this.config.enableAutomaticLLM) {
+          this.updateLLMConnection().catch((error) => {
+            LoggerProxy.logger.error('Meeting:index#join --> Update LLM Connection Failed', error);
+
+            Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.LLM_CONNECTION_AFTER_JOIN_FAILURE, {
+              correlation_id: this.correlationId,
+              reason: error?.message,
+              stack: error.stack,
+            });
+          });
+        }
+
+        return join;
+      })
+      .then((join) => {
+        if (isBrowser) {
+          // @ts-ignore - config coming from registerPlugin
+          if (this.config.receiveTranscription || options.receiveTranscription) {
+            if (this.isTranscriptionSupported()) {
+              LoggerProxy.logger.info(
+                'Meeting:index#join --> Attempting to enabled to recieve transcription!'
+              );
+              this.receiveTranscription().catch((error) => {
```

Comment:
receiveTranscription() will get called before updateLLMConnection() resolved. Previously the call to updateLLMConnection() was awaited. Is this OK?

Reply:
I have confirmed with @arun3528 that transcription (at least this implementation), does not rely on llm and has creates its own websocket connectino

Reply:
we are good if both are not enabled  at the same time